lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
+  "env": {}
```

Comment:
@josepjaume Thanks man but I already added the addons but Github is not smart enough to point at them...